src/lib/helpers.ts

```diff
@@ -2,7 +2,7 @@ export function exponentialBackoff(n: number, base = 1000): number {
     return Math.pow(2, n) * base + Math.floor(Math.random() * base);
 }
 
-export function parseJSON(json: string): object | any[] {
+export function parseJSON(json: string): object | any[] | null {
```

Comment:
should probable be `UnknownDictionaryKnownValues | null`

Reply:
Will fix